bot/custom.py

```diff
@@ -81,7 +81,7 @@ async def on_submit(self, interaction: discord.Interaction):
         # Format confirmation to the user
         output_message = f"New command added! You can use it like `{CMD_PREFIX}{name}`. "
 
-        await send_message(f"{str(interaction.user)} has changed the `{name}` command", client.log)
+        await send_message(f"{str(interaction.user)} has changed the `{name}` command. New content: ```{response.replace('`','\\`')}```", client.log)
```

Comment:
The one caveat here is that if this message + the new response is over 2000 characters, it'll be split into two so the \`\`\` won't render properly. I think I'm still fine with it though, that'll only be in rare instances and I don't think it's worth the trouble of handling it.